run_tests.py

```diff
@@ -69,6 +72,9 @@
             else:
                 shutil.copy(file_or_dir_src, dest)
```

Comment:
This ends up copying `config_version.txt` to `machine_name/configurations/configurations` it should actually live in `machine_name/configurations` otherwise the upgrade script won't find it.